### ISSUE TITLE
[Feature] Trigger events in automation apps when timer is started/stopped

### ISSUE THREAD

**Issue by grooveboxunited** [Razeeman/Android-SimpleTimeTracker#76]
There is already [support for automatically starting and stopping activities](https://github.com/Razeeman/Android-SimpleTimeTracker/issues/11) using automation apps like Tasker, and this works great. It would be nice to also be able to go the other way - i.e. use Simple Time Tracker to trigger other events on your device.

For example say I have an activity "Driving". When I start this activity I want to tell an app like Tasker to open my maps app, and set my phone to Do Not Disturb.

No idea if this is easily achievable but thought I'd suggest it.

Thanks for this great app!

**Comment by Razeeman** [comment]
Hello! Thank you for the suggestion. I've added a setting to also send events to automated tracking apps and add information about parameters into the Automated tracking info dialog. Basically extras will be the same as in receiving events (activity name, comment and a tag name) and actions will be "com.razeeman.util.simpletimetracker.EVENT_STARTED_ACTIVITY" and "com.razeeman.util.simpletimetracker.EVENT_STOPPED_ACTIVITY". Tested with macrodroid app. I can make a debug version if you want to check how it will work. It will be in the next release.
